recipes/orekit/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "orekit" %}
+{% set version = "9.0" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://www.orekit.org/git/orekit-labs/python-wrapper-main.git
+  git_tag: orekit-v9
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win32 or linux32]
+
+  number: 0
+
+  # script: python setup.py install --single-version-externally-managed --record record.txt
+
+  rpaths:  # Only valid for linux
+    - lib/
+    - jre/lib/amd64/
+    - jre/lib/amd64/server/
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - jcc >=3
+    - openjdk 8.0.121 1  # Force anaconda openjdk
```

Comment:
Please move comment to line above. Also pinning to the build number is not great. What is wrong with `openjdk`?

Reply:
Comment is moved. The pinning is to force using anaconda openjdk. There is currently an issue with the conda-forge openjdk https://github.com/conda-forge/openjdk-feedstock/issues/12